interfaces/cython/cantera/yaml2ck.py

```diff
@@ -0,0 +1,956 @@
+#!/usr/bin/env python3
+# encoding: utf-8
+
+# This file is part of Cantera. See License.txt in the top-level directory or
+# at https://cantera.org/license.txt for license and copyright information.
+
+# Portions of this file are licensed from Frhodo 
+#     (https://github.com/Argonne-National-Laboratory/Frhodo).
+# Those portions are copyright 2020 UChicago Argonne, LLC
+# Redistribution and use in source and binary forms, with or without modification, 
+#     are permitted provided that the following conditions are met:
+# 1. Redistributions of source code must retain the above copyright notice, this 
+#     list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright notice, this 
+#     list of conditions and the following disclaimer in the documentation and/or 
+#     other materials provided with the distribution.
+# 3. Neither the name of the copyright holder nor the names of its contributors may 
+#     be used to endorse or promote products derived from this software without 
+#     specific prior written permission.
+
+# Portions of this file are licensed from pyMARS 
+#     (https://github.com/Niemeyer-Research-Group/pyMARS).
+# Those portions are copyright 2016-2019 Parker Clayton, Phillip Mestas, 
+#     and Kyle Niemeyer
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this
+#     software and associated documentation files (the "Software"), to deal in the 
+#     Software without restriction, including without limitation the rights to use, 
+#     copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the 
+#     Software, and to permit persons to whom the Software is furnished to do so, 
+#     subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies 
+#     or substantial portions of the Software.
+
+"""
+yaml2ck.py: Convert Cantera YAML input files or solution objects to 
```

Comment:
```suggestion
yaml2ck.py: Convert Cantera YAML input files or `Solution` objects to
```

If `Solution` objects are supported, it'd be helpful to show how to do that. However, it may not be appropriate to document that in the CLI docstring, but rather elsewhere.